### DISCUSSION TITLE
async method lacks 'await' operators and will run synchronously

### DISCUSSION THREAD

**Discussion by patagoniahiker** [Dotnet-Boxed/Templates#78] [Q&A]
1>c:\RehanBoiler\StudentsSample_V2\Services\Feed\FeedService.cs(181,51,181,59): warning CS1998: This async method lacks 'await' operators and will run synchronously. Consider using the 'await' operator to await non-blocking API calls, or 'await Task.Run(...)' to do CPU-bound work on a background thread.

**Comment by RehanSaeed** [comment]
Thanks for raising this. This warning is intentional, it's up to you to fill in that method (most likely using an await statement somewhere) and then the warning will go away.
